### PR TITLE
Changes 1

### PR DESCRIPTION
**Pull request by gregk27**
Made tweaks (+renamed) autopage

### CODE REVIEW

File: app/src/main/res/layout-sw600dp/pregamepage.xml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            android:layout_alignParentTop="true">
+
+            <TableRow
+                android:id="@+id/startPosition"
+                android:layout_width="fill_parent"
+                android:layout_marginBottom="17dp"
+                android:weightSum="4">
+
+                <TextView
+                    android:id="@+id/pregame0"
```
Comment (ajayyy) [inline app/src/main/res/layout-sw600dp/pregamepage.xml]: It's probably a good idea to get this id changed to something more descriptive at some point, though it does not matter much as it's just a textview.

File: app/src/main/res/layout-sw600dp/pregamepage.xml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                    <RadioButton
+                        android:id="@+id/hab2"
+                        android:layout_width="wrap_content"
+                        android:layout_height="match_parent"
+                        android:layout_alignTop="@+id/tableLayout"
+                        android:layout_alignParentStart="true"
+                        android:layout_alignParentLeft="true"
+                        android:layout_weight="1"
+                        android:text="HAB 2"
+                        android:textSize="@dimen/text_small" />
```
Comment (ajayyy) [inline app/src/main/res/layout-sw600dp/pregamepage.xml]: Maybe "Level 1" and "Level 2".

I think we should record if it starts on the left, middle or right in a spinner along with which level.

**Review by ajayyy — COMMENTED** [review]
Missing "Left hab area" (where would we even place this though anyway).

Also, we should move the ratings bars to the qualitative page.

**Comment by ajayyy** [comment]
Also, you broke the "Robots Carried" on the post game page I think. I think it is just due to confusion between table layout (what we used before), and grid layout (what Xan changed it to).

File: app/src/main/res/layout-sw600dp/pregamepage.xml (hunk 1 above)
Comment (gregk27) [reply app/src/main/res/layout-sw600dp/pregamepage.xml]: I might just remove it outright

File: app/src/main/res/layout-sw600dp/pregamepage.xml (hunk 2 above)
Comment (gregk27) [reply app/src/main/res/layout-sw600dp/pregamepage.xml]: I'll look into it

**Comment by gregk27** [comment]
That layout was a straight copy-paste so IDK why that happened

**Comment by ajayyy** [comment]
Oh, it could just be a rendering error then, sorry.

**Comment by gregk27** [comment]
And I'm not bothering with left hab area
If that happens it can be noted elsewhere

**Comment by gregk27** [comment]
And I think there is a layout issue, just IDK what

File: app/src/main/res/layout-sw600dp/pregamepage.xml (hunk 1 above)
Comment (ajayyy) [reply app/src/main/res/layout-sw600dp/pregamepage.xml]: It probably was just there because of us using a relative layout at some point.

**Comment by ajayyy** [comment]
Looks good, though I do think we should revert to Xan's layout on the pregame page due to the better margins plus the checkboxes for the preload object (since if you accidentally tap one, and they are not preloaded, it could be bad). The code forces you to only select maximum one though.

I'm merging it back in, I'll make a pull request to your fork.

**Comment by ajayyy** [comment]
See https://github.com/Aree-Vanier/LakeEffectScoutingApp/pull/13

**Comment by ajayyy** [comment]
Also, I've done further testing and can confirm that any layout issues on the post game page are present in [x3a1n4/master](https://github.com/x3a1n4/LakeEffectScoutingApp/tree/master) as well and not from you.

**Comment by gregk27** [comment]
~@ajayyy they HAVE to be preloaded, so we can only have 2~
Misread.

**Pull request merged**